libcontainer/mount_linux.go

```diff
@@ -10,8 +10,9 @@ import (
 type mountError struct {
```

Comment:
Commenting here as I can't comment on github commit description. In the commit you showed how it looked like before, but can you show how it looks now?

I can't think of a simple way add a test for a clear error too that will not need a refactor every time we change this. Probably it is not worth it adding a test.

Reply:
Commit amended to show both before and after error message.

Reply:
Thanks!